core/src/com/unciv/ui/screens/pickerscreens/ImprovementPickerScreen.kt

```diff
@@ -144,7 +152,14 @@ class ImprovementPickerScreen(
             }
 
             val explanationText = when {
-                proposedSolutions.any() -> proposedSolutions.joinToString("}\n{", "{", "}").toLabel()
+                proposedSolutions.any() -> {
+                    val text = proposedSolutions.joinToString("}\n{", "{", "}")
+                    if (removalImprovement != null && proposedSolutions.size == 1)
+                        "$text\n{Pick now!}".toLabel().onClick {
+                            accept(removalImprovement, improvement)
+                        }
+                    else text.toLabel()
```

Comment:
Are we able to center this text so that it looks more in line with the pick now?